### PR TITLE
[SDP-200] Landing Page for Org Select (serveday.com)

### PR DESCRIPTION
**Pull request by freakinhuge**
Landing Page for Org Select (serveday.com). Adds landing page to home/show. Constrains non-admin routes to organization subdomain. Adds message if no next_event for organization (probably need to update what's shown).

### CODE REVIEW

File: app/models/organization.rb [hunk 1]
```diff
@@ -108,6 +108,12 @@ def self.policy_class
     Admin::OrganizationPolicy
   end
 
+  def city_state
+    if self.city.present? && self.state.present?
+      "#{self.city.titleize}, #{Carmen::Country.coded("us").subregions.named(self.state).code}"
```
Comment (cjarrett) [inline app/models/organization.rb]: are we assuming states have a certain format? if so, we should validate that on model save.

Comment (cjarrett) [reply app/models/organization.rb]: also, just thought about this but let's not allow login from here...remove nav icon and replace with home that goes to serveday.com

Comment (freakinhuge) [reply app/models/organization.rb]: Yes states should have a certain format b/c they are [picked from a dropdown](https://github.com/highlands/serveapp/blob/master/app/views/admin/organizations/_form.html.haml#L52-L55). Unless we are allowing the new organization form somewhere else to just be a text input?

Comment (cjarrett) [reply app/models/organization.rb]: nope, should be a dropdown everywhere. let's enforce that on the model..only other thing I ran into is @organization.name in the navbar_brand doesn't show name if no logo. Maybe reference current_organization there?

Comment (freakinhuge) [reply app/models/organization.rb]: We would need to add in a country attribute to organization and drop-downs accordingly. right?

Comment (cjarrett) [reply app/models/organization.rb]: the attribute is already there, but yes on drop downs.

**Pull request merged**